Server-errors-and-solutions.md

```diff
@@ -0,0 +1,80 @@
+This document talks about different types of server problems that have been fixed before, and it suggests ways to stop those problems from happening again. The types of problems include issues with data, problems when lots of people are using the server, situations that don't happen very often, making sure the server works with old internet browsers, mistakes in the way the server thinks, problems when the internet is slow, and errors that don't need to be written down. For each type of problem, it gives an example and tips on how to stop it from happening.
```

Comment:
document --> page

talks about --> discusses

I don't understand parts of the second sentence ("errors that don't need to be written down", "mistakes in the way the server thinks"). I suggest you just get rid of that sentence altogether.

Reply:
Yes, please do that.

Reply:
done